Plugins/SequencePlugin/Source/SequencePlugin/Public/Util/SequenceSupport.h

```diff
@@ -139,6 +140,70 @@ class SEQUENCEPLUGIN_API USequenceSupport : public UObject
 		FNameId(TEXT("blastsepolia"),168587773),
 		FNameId(TEXT("xaisepolia"),37714555429)};
 	
+	static inline TMap<ENetwork, int64> NetworkEnumToIdMap = {
+		{ENetwork::Ethereum, 1},
+		{ENetwork::Optimism, 10},
+		{ENetwork::BNBSmartChain, 56},
+		{ENetwork::GnosisChain, 100},
+		{ENetwork::PolygonNetwork, 137},
+		{ENetwork::PolygonZkEVM, 1101},
+		{ENetwork::AstarZkEVM, 3776},
+		{ENetwork::BaseCoinbase, 8453},
+		{ENetwork::Base, 8453},
+		{ENetwork::OasysHomeverse, 19011},
+		{ENetwork::ArbitrumOne, 42161},
+		{ENetwork::ArbitrumNova, 42170},
+		{ENetwork::Avalanche, 43114},
+		{ENetwork::Blast, 81457},
+		{ENetwork::XAI, 660279},
+		{ENetwork::BNBSmartChainTestnet, 97},
+		{ENetwork::B3Sepolia, 1993},
+		{ENetwork::XrSepolia, 2730},
+		{ENetwork::ApeChainTestnet, 33111},
+		{ENetwork::OasysHomeverseTestnet, 40875},
+		{ENetwork::AvalancheTestnet, 43113},
+		{ENetwork::PolygonAmoy, 80002},
+		{ENetwork::BaseSepolia, 84532},
+		{ENetwork::ArbitrumSepolia, 421614},
+		{ENetwork::AstarZkYotoTestnet, 6038361},
+		{ENetwork::Sepolia, 11155111},
+		{ENetwork::OptimismSepolia, 11155420},
+		{ENetwork::BlastSepolia, 168587773},
+		{ENetwork::XaiSepolia, 37714555429}
+		};
+
+	static inline TMap<ENetwork, FString> NetworkEnumToNameMap = {
+	{ENetwork::Ethereum, TEXT("ethereum")},
+	{ENetwork::Optimism, TEXT("optimism")},
+	{ENetwork::BNBSmartChain, TEXT("bnbsmartchain")},
+	{ENetwork::GnosisChain, TEXT("gnosischain")},
+	{ENetwork::PolygonNetwork, TEXT("polygon")},
+	{ENetwork::PolygonZkEVM, TEXT("polygonzkevm")},
+	{ENetwork::AstarZkEVM, TEXT("astarzkevm")},
+	{ENetwork::BaseCoinbase, TEXT("base(coinbase)")},
+	{ENetwork::Base, TEXT("base")},
+	{ENetwork::OasysHomeverse, TEXT("oasyshomeverse")},
+	{ENetwork::ArbitrumOne, TEXT("arbitrumone")},
+	{ENetwork::ArbitrumNova, TEXT("arbitrumnova")},
+	{ENetwork::Avalanche, TEXT("avalanche")},
+	{ENetwork::Blast, TEXT("blast")},
+	{ENetwork::XAI, TEXT("xai")},
+	{ENetwork::BNBSmartChainTestnet, TEXT("bnbsmartchaintestnet")},
+	{ENetwork::B3Sepolia, TEXT("b3sepolia")},
+	{ENetwork::XrSepolia, TEXT("xrsepolia")},
+	{ENetwork::ApeChainTestnet, TEXT("apechaintestnet")},
+	{ENetwork::OasysHomeverseTestnet, TEXT("oasyshomeversetestnet")},
+	{ENetwork::AvalancheTestnet, TEXT("avalanchetestnet")},
+	{ENetwork::PolygonAmoy, TEXT("polygonamoy")},
+	{ENetwork::BaseSepolia, TEXT("basesepolia")},
+	{ENetwork::ArbitrumSepolia, TEXT("arbitrumsepolia")},
+	{ENetwork::AstarZkYotoTestnet, TEXT("astarzkyototestnet")},
+	{ENetwork::Sepolia, TEXT("sepolia")},
+	{ENetwork::OptimismSepolia, TEXT("optimismsepolia")},
+	{ENetwork::BlastSepolia, TEXT("blastsepolia")},
+	{ENetwork::XaiSepolia, TEXT("xaisepolia")}
+	};
```

Comment:
Should these match the display names?

Reply:
yep. i got the names from the already existing <id,name> map. the display name just shows them nicer with caps and spaces